stumpy/motifs.py

```diff
@@ -240,6 +248,18 @@ def motifs(
         The p-norm to apply for computing the Minkowski distance. This parameter is
         ignored when `normalize == True`.
 
+    T_subseq_isconstant : numpy.ndarray or function, default None
+        A boolean array that indicates whether a subsequence in `T` is constant
+        (True). Alternatively, a custom, user-defined function that returns a
+        boolean array that indicates whether a subsequence in `T` is constant
+        (True). The function must only take two arguments, `a`, a 1-D array,
+        and `w`, the window size, while additional arguments may be specified
+        by currying the user-defined function using `functools.partial`. Any
+        subsequence with at least one np.nan/np.inf will automatically have its
+        corresponding value set to False in this boolean array. Note that it is
+        the responsibility of user to ensure that the provided matrix profile `P`
+        is computed while considering `T_subseq_isconstant`.
```

Comment:
Not sure if the last sentence above is a good addition to the description of param `T_subseq_isconstant`

Reply:
I don't understand what that sentence is trying to convey :(

Reply:
`mateix profile (P)` is one of the parameters of function `motifs`. So, the user needs to compute it and pass  it to `motifs`. However, the user needs to be careful and conside the `T_subseq_isconstant` in the computation of matrix profile if they would like to pass  `T_subseq_isconstant` to `motifs` . 

In other words, if `T_subseq_isconstant` is not None, then the following code is not correct::
```
# T_subseq_isconstant is not None

mp = stumpy.stump(T, m)
P = mp[:, 0].astype(np.float64)
out = motifs(T, P, ..., T_subseq_isconstant=T_subseq_isconstant)
```

The correct way is:
```
# T_subseq_isconstant is not None

mp = stumpy.stump(T, m, T_A_subseq_isconstant=T_subseq_isconstants)
P = mp[:, 0].astype(np.float64)
out = motifs(T, P, ..., T_subseq_isconstant=T_subseq_isconstant)
```

Note that I am passing `T_subseq_isconstants` to function `stump` as well.

Reply:
> In other words, if T_subseq_isconstant is not None, then the following code is not correct:

Is it incorrect only if `T_subseq_isconstant` is some custom function or custom array (that is different from what is produced by `core.rolling_isconstant`)? If so, I don't feel like the last sentence is necessary. To some extent, it is similar to when an exclusion zone is specified and I think that's an advance feature.

Reply:
> To some extent, it is similar to when an exclusion zone is specified and I think that's an advance feature.

I see your point! I will remove the last sentence.